docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3.8'
+
+services:
+  agent:
+    image: ghcr.io/virattt/ai-hedge-fund:main

```

Comment:
@charlesvien thanks for making the PR. Could we switch to use build directive here so that contributors/anyone can build the image locally without needing registry access? e..g,:
```
    build:
      context: .
      dockerfile: Dockerfile
```

Reply:
Perhaps there should be a separate docker-compose-dev.yml with the build directive for contributors ?
This docker compose for the users who want to use the docker images, but not looking to make any changes / build the source.